README.md

```diff
@@ -1 +1,44 @@
-# Project1
\ No newline at end of file
+# Project1
+Hello Microverse
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. The goal here is to master all of the tools and best practices I learned . All about gitflow , linters.
+
+set up a new repository and prepare it for development using best practices.
+Built With
+
+    HTML
+    CSS
+    HTML & CSS Linter
+
+Getting Started
+
+Please get your browser update. To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+git clone in your terminal
+cd hello-microverse
+start index.html
+Authors
+
+👤 Author
+
+    GitHub: @VSeray
```

Comment:
- [ ] I can see in the author section that there are no links provided kindly follow the process the first review suggested. like Twitter : [your name or xUsername](https//the link of your Twitter username). This will help people who will visit this Repository to get the link of your contact if they liked the project.